cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -320,6 +320,7 @@ protected static CConfiguration createBasicCConf() throws IOException {
     CConfiguration cConf = CConfiguration.create();
     cConf.set(Constants.Service.MASTER_SERVICES_BIND_ADDRESS, InetAddress.getLoopbackAddress().getHostAddress());
     cConf.set(Constants.CFG_LOCAL_DATA_DIR, tmpFolder.newFolder("data").getAbsolutePath());
+    cConf.set(Constants.SupportBundle.LOCAL_DATA_DIR, tmpFolder.newFolder("supportBundleData").getAbsolutePath());
```

Comment:
nit: Let's just call this folder `data` since the parent folder is already support-bundle-specific.